### PR TITLE
Update translations: ruby-core and manual instructions (id)

### PR DESCRIPTION
**Pull request by meisyal**
As mentioned on #157, I found that community page `id` is outdated. These changes contains translation improvements and updated contents. Updated contents mean the content synchronizes with `en` pages. Please, review this PR.

Thank you.

### CODE REVIEW

**Comment by meisyal** [comment]
cc: @gozali

**Comment by meisyal** [comment]
Help me to review the translation, please, @gozali.

**Comment by meisyal** [comment]
Any description should be added for "Ruby Indonesia on Slack"? e.g.  Join id-ruby slack chatroom! or? @gozali

**Comment by gozali** [comment]
yes, something like that

File: id/community/ruby-core/index.md [hunk 1]
```diff
@@ -4,141 +4,168 @@ title: "Ruby Core"
...
-milis. Anda diharapkan untuk berpartisipasi aktif dalam diskusi
-tersebut.
+Tim Ruby Core merawat [bug tracker][6] untuk memasukkan *patch* dan laporan
+*bug* ke Matz dan rekan-rekannya. Laporan-laporan ini juga dimasukkan ke
+[milis Ruby-Core][mailing-lists] untuk didiskusikan, jadi Anda bisa
+mengkonfirmasi terlebih dahulu apakah permintaan Anda akan diperhatikan atau
+tidak. Anda juga bisa mengirim *patch* langsung ke milis. Anda diharapkan
+untuk berpartisipasi aktif dalam diskusi tersebut.
+
+Mohon lihat [Patch Writer's Guide][writing patches] untuk beberapa tips,
```
Comment (stomar) [inline id/community/ruby-core/index.md]: writing-patches

Comment (meisyal) [reply id/community/ruby-core/index.md]: Thanks for catching that. Just, fixed that line.

**Comment by meisyal** [comment]
I just added Slack chatroom link of Indonesian Ruby community under "Komunitas Pengguna Ruby Indonesia", @gozali.

**Comment by gozali** [comment]
+1 thanks @meisyal

**Comment by meisyal** [comment]
Is this ready, @stomar? Any concern?

**Comment by hsbt** [comment]
@meisyal @gozali Thanks always. I will merge your contribution.

**Pull request merged**